Dockerfile.r

```diff
@@ -0,0 +1,13 @@
+FROM rpy2/rpy2:2.9.x
+
+ADD . /gluonts
+
+RUN pip3 install /gluonts[shell]
```

Comment:
We could also just suggest to pip install gluonts from PyPi.

Reply:
You're right. Sounds better